composer.json

```diff
@@ -1,111 +1,111 @@
 {
     "name": "catchdigital/drucker",
     "description": "Docker + drupal composer stack",
-    "version": "1.0.0",
+    "version": "2.0.0",
     "type": "project",
     "license": "MIT",
     "minimum-stability": "dev",
     "config": {
-      "platorm": {
-        "php": "7.2.8"
-      }
+        "platorm": {
+            "php": "7.4.4"
+        }
     },
     "authors": [
-      {
-        "name": "Alberto Contreras",
-        "email": "a.contreras@catchdigital.com"
-      }
+        {
+            "name": "Alberto Contreras",
+            "email": "a.contreras@catchdigital.com"
+        },
+        {
+            "name": "Pierre Villette",
+            "email": "p.villette@catchdigital.com"
+        }
     ],
     "repositories": [
-      {
-        "type": "composer",
-        "url": "https://packages.drupal.org/8"
-      }
+        {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        }
     ],
     "require": {
-        "composer/installers": "^1.2",
-        "drupal-composer/drupal-scaffold": "^2.2",
-        "cweagans/composer-patches": "~1.0",
-        "drupal/core": "^8.6",
-        "drush/drush": "~9",
-        "drupal/console": "~1.0",
-        "vlucas/phpdotenv": "^2.4"
-    },
-    "require-dev": {
+        "composer/installers": "^1.9",
+        "drupal-composer/drupal-scaffold": "^2.6",
+        "cweagans/composer-patches": "^1.6",
+        "drush/drush": "^10.2",
+        "vlucas/phpdotenv": "^5.0"
```

Comment:
It looks like core and core-recommended are both not required anymore. I think we need to add "drupal/core-recommended:^9.0"